### DISCUSSION TITLE
Join the PeerJS Team: We're Looking for New Maintainers!

### DISCUSSION THREAD

**Discussion by jonasgloning** [peers/peerjs#1173] [Announcements]
**Update: both @psanders and @irgalamarr have joined the maintainer team 🎉🥳**

---
Dear PeerJS Community,

First of all: PeerJS remains working, supported, and I will continue to keep the server up and running. I will also do my best to fix all remaining bugs for now.

However, it has become clear that PeerJS deserves more time and dedication than I am currently able to provide. There are more bugs to fix, issues to answer, IPs to block, and code to document than I can manage at this time. Therefore, I am reaching out to the community in search of new potential maintainers.

**I need help with:**
1. **Issue Management:** Responding to, triaging, and managing GitHub issues and pull requests. That’s a surprising amount of work.
2. **Documentation:** Creating and updating documentation to help users and contributors. Great documentation would for sure make 1 easier.
3. **Development:** Working on the client and server-side code to improve and expand PeerJS's capabilities. There’s an infinite amount of work possible here, but I think it’s the least of the priorities. The code works well, and it is likely to stay useful at this level without much intervention.

My goal is to support PeerJS now and in the future, but I am prepared to step aside to make room for those more capable or who can dedicate more time and resources to the project.

I took over the project in a hurry, without a lot of guidance from or contact with previous maintainers. I am committed to doing things differently, ensuring a smooth transition process, and will remain involved a while to ensure the new maintainers are well-equipped to lead the project forward.

If you are passionate about PeerJS/Javascript and feel that you have the enthusiasm to take on a maintainer role, please reach out by commenting on this issue or contacting me directly at [scheitel.trend0o@icloud.com](mailto:scheitel.trend0o@icloud.com).

Thank you for your continued support and contributions to PeerJS.

Best,
Jonas

**Comment by psanders** [comment]
Hi @jonasgloning,

I am passionate about this problem space.

I would love to chat about this announcement.

**Comment by psanders** [comment]
I tried to join the Telegram community, as mentioned on the main repository, but the link had expired.

**Comment by cherishsantosh** [comment]
I am interested in maintainer role

**Reply by jonasgloning** [reply]
That's great to hear! I've send you an email to the address linked in your Github profile.

**Comment by irgalamarr** [comment]
Hi @jonasgloning! I would love to help.
I have 4 years experience to maintain similar but commercial product.

**Reply by jonasgloning** [reply]
Very glad to read that @irgalamarr!

Please send me an email to scheitel.trend0o@icloud.com, so we can continue this discussion privately. I would be happy to schedule a quick Zoom call in the coming days – please include some 
suggested dates in your email.

**Comment by GeorgeHcc** [comment]
hi, @jonasgloning! sounds great to maintain this project!
I am working in multi-person video conference based on webrtc, and wish peerjs more and more powerful .
